libs/uberz/include/uberz/ReadableArchive.h

```diff
@@ -30,8 +30,13 @@ namespace filament::uberz {
 // offset fields into pointers.
 void convertOffsetsToPointers(struct ReadableArchive* archive);
 
+#if defined(__clang__)
 #pragma clang diagnostic push
 #pragma clang diagnostic warning "-Wpadded"
+#elif defined(_MSC_VER)
+#pragma warning( push )
+#pragma warning(disable : 4324)
```

Comment:
Add a comment describing the warning please.

Reply:
Is this still applicable after fb7c465c5d06411b2845e02397c98f8fa8493d74?

Reply:
No, looks good :)